api/v1beta1/kyma_types.go

```diff
@@ -420,3 +422,28 @@ func (kyma *Kyma) AllModulesReady() bool {
 	}
 	return true
 }
+
+func (kyma *Kyma) SyncEnabled() bool {
+	syncVal, found := kyma.GetLabels()[SyncLabel]
```

Comment:
1. the labels might be nil, nil pointer should be detected first.
2. in this PR, it's better the value of `kyma.sync` field still respect.